src/main/java/com/ccommit/auction_server/model/ELK/DocumentBid.java

```diff
@@ -0,0 +1,35 @@
+package com.ccommit.auction_server.model.ELK;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.elasticsearch.annotations.*;
+
+import java.time.LocalDateTime;
+
+
+@Document(indexName = "bid_index")
+@Getter
+@Setter
+@Setting(settingPath = "/static/elastic/elastic-settings.json")
+@Mapping(mappingPath = "/static/elastic/product-mappings.json")
+public class DocumentBid {
+    @Id
+    private String id;
+
+    @Field(type = FieldType.Long, name = "bid_id")
+    private Long bidId;
+
+    @Field(type = FieldType.Long, name = "buyer_id")
+    private Long buyerId;
+
+    @Field(type = FieldType.Long, name = "product_id")
+    private Long productId;
+
+    @Field(name = "bid_time", pattern = "uuuu-MM-dd'T'HH:mm:ss.SSS", format = {})
```

Comment:
format = {} 으로 설정하신 이유를 댓글로 달아주세요

Reply:
해당하는 내용은 elasticsearch 버전이 올라감에 있어 DateFormat.custome이 사라지게 되면서 빈값으로 넣어야 커스텀하게 설정할수 있어 format = {} 으로 설정하였습니다. 아래는 해당하는 내용입니다.

Values based on Elasticsearch reference documentation . The patterns are taken from this documentation and slightly adapted so that a Java java.time.format.DateTimeFormatter produces the same values as the Elasticsearch formatter. Use format = {} to disable built-in date formats in the Field annotation. If you want to use only a custom date format pattern, you must set the format property to empty {}.

Reply:
빈값으로 넣어야 커스텀하게 설정 -> 이 부분을 예시를 들어가며 구체적으로 설명 부탁드립니다.

Reply:
위의 영어 본문을 해석한 내용입니다. 
Elasticsearch 참조 문서에 기반한 값입니다. 패턴은 이 문서에서 가져온 것으로 Java.time.format.DateTimeFormatter가 Elasticsearch 포맷터와 동일한 값을 생성합니다. Field 주석에서 기본 제공 날짜 형식을 사용하지 않도록 설정하려면 형식 = {}을(를) 사용하십시오. 사용자 지정 날짜 형식 패턴만 사용하려면 형식 속성을 빈 {}으로 설정해야 합니다.
[출처 : [JAVA](spring-data-elasticsearch:5.1.1) DateFormat.java 주석]

@Field(name = "start_time", pattern = "uuuu-MM-dd'T'HH:mm:ss.SSS", format = {})
위의 예시와 같이 DateFormat에서 제공하는 패턴이 아닌 Custom 한 패턴을 사용하고자 하여 format = {} 으로 구성하였습니다.

전 버전의 Elasticsearch의 format 값으로 DateFormat.custom이 있었으나 현재 버전에서는 없어졌습니다.